alerta/utils/config.py

```diff
@@ -88,6 +88,9 @@ def get_user_config():
         if 'ALLOWED_GITLAB_GROUPS' in os.environ:
             config['ALLOWED_OIDC_ROLES'] = os.environ['ALLOWED_GITLAB_GROUPS'].split(',')
 
+        if 'LDAP_TLS_ALLOW_SELFSIGNED' in os.environ:
+            config['LDAP_TLS_ALLOW_SELFSIGNED'] = True if os.environ['LDAP_TLS_ALLOW_SELFSIGNED'] == 'True' else False
+
```

Comment:
Please remove this as it's not needed. No other LDAP settings can be set by environment variable.

Reply:
I really wanted it to be an environment variable for some reason. It's deleted.